Stellarium/stellarium
# Restore drawing of orbits on open KeplerOrbits.

### Description
It seems that a change in the orbit code in 2021 suppressed drawing of Kepler orbits.
This should fix it. Objects on Kepler orbits are drawn orbit_good days around the epoch date. 

Another thing to fix here is the removal of the entry "orbit_visualization_period" in ssystem_*.ini. 
For closed orbits with special functions, this should be equal to orbit_period. 
For Kepler orbits, this is governed by orbit_good, or orbital period computed from the elements. 

Fixes # (issue)

### Screenshots (if appropriate):

### Type of change


### How Has This Been Tested?




- Check that e.g. Hyakutake (C/1996 B1) and Hale-Bopp (C/1995 O1) show their inner orbits. 
- Load 1000_comets.ini and use SolarSystemObserver to see numerous comet orbits flash in and out of visibility as comets come and go in rapid timelapse. 

**Test Configuration**:
* Operating system: Windows 11
* Graphics Card: Geforce (irrelevant)

## Checklist:
